introduction.md

```diff
@@ -2,3 +2,6 @@ Myself Chaitramani S.
 I am from Banglore.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?